### PR TITLE
Releasing ClickToView for third party embedded content 

### PR DESCRIPTION
**Pull request by frj**
This PR removes the code that implements the AB testing for the ClickToView component.

This essentially releases the feature that requires users to provide consent to being tracked by third parties before they can see the embedded content they provide. 

For more details see:

- https://github.com/guardian/dotcom-rendering/pull/2408
- https://github.com/guardian/dotcom-rendering/pull/2474
- https://github.com/guardian/dotcom-rendering/pull/2525

### CODE REVIEW

File: cypress/integration/parallel-2/article.elements.spec.js [hunk 1]
```diff
@@ -109,19 +109,30 @@ describe('Elements', function () {
...
 					.should('not.be.empty')
 					.then(cy.wrap);
 			};
+			const getClickToViewOverlay = () => {
+				return cy
+					.get('div[data-cy="click-to-view"]')
+					.its('0')
+					.should('not.be.empty')
+					.then(cy.wrap);
+			};
```
Comment (oliverlloyd) [inline cypress/integration/parallel-2/article.elements.spec.js:126]: We only need this pattern for iframes (which are treated differently in Cypress). To just click an element you can simply use `cy.get('button[data-cy=click-to-view-button]').click()` and the contains can just be `cy.contains('hosted on wnyc.org');` which checks the whole document.

Or you could (I'm fairly sure) chain them to restrict the contains like:

```
cy.get('button[data-cy=click-to-view-button]').contains('hosted on wnyc.org').click()
```

File: src/web/components/ClickToView.tsx [hunk 2]
```diff
@@ -209,6 +196,7 @@ export const ClickToView = ({
 						icon={<SvgCheckmark />}
 						iconSide="left"
 						onClick={() => handleClick()}
+						data-cy="click-to-view-button"
```
Comment (oliverlloyd) [inline src/web/components/ClickToView.tsx:199]: As above, we can remove these

**Review by oliverlloyd — APPROVED** [review]
Looks great!

Just a small one around simplifying the Cypress test

File: src/web/components/ClickToView.tsx (hunk 2 above)
Comment (frj) [reply src/web/components/ClickToView.tsx:199]: I think we need this one, but I can remove the ```data-cy="click-to-view"``` on the component.

Comment (oliverlloyd) [reply src/web/components/ClickToView.tsx:198]: Ah, yes, you're right, they're different 👍

**Pull request merged**